crates/orderbook/openapi.yml

```diff
@@ -1661,6 +1661,9 @@ components:
                 $ref: "#/components/schemas/UID"
               executedAmount:
                 $ref: "#/components/schemas/BigUint"
+        isWinner:
+          type: boolean
+          description: whether the solution is a winner or not
```

Comment:
the description does not provide context above the obvious. Would be more helpful if it said what it means to be a winner (i.e. received the right to get executed).